sbin/repair_services.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# shellcheck enable=all
+
+#### This script repairs core services that are not currently running
+
+repair_services() {
+    services=(
+        "nginx"
+        "apache2"
+        "postgresql"
+        "nullsmtpd"
+        "submitty_websocket_server"
+        "submitty_autograding_worker"
+        "submitty_autograding_shipper"
+        "submitty_daemon_jobs_handler"
+    )
+
+    for service in "${services[@]}"; do
+        status=$(sudo systemctl status "${service}")
+        if ! echo "${status}" | grep -q 'Active: active'; then
```

Comment:
```suggestion
       if ! sudo systemctl is-active --quiet "${service}"; then
```
Manually greping the output seems flaky, `systemctl is-active` will return the result we want directly as an error code